### DISCUSSION TITLE
Terser

### DISCUSSION THREAD

**Discussion by esseifforte** [ixkaito/bathe#16] [General]
Cab you add terser plugin webpack?

thank you in advance

**Comment by ixkaito** [comment]
@esseifforte Thanks. Any reason to use terser?

**Reply by esseifforte** [reply]
just to understand, that's all, considering that many sites, after having retired uglifyjs, use it to minimize js
it is absolutely not a criticism, indeed.
your repo is cool. thanks.

**Reply by ixkaito** [reply]
Thanks! I'll consider it.
